server.js

```diff
@@ -1,12 +1,23 @@
-// server.js
+import express from 'express';
+import dataUriToBuffer from 'data-uri-to-buffer';
+import bodyParser from 'body-parser';
+import svgToDataURL from 'svg-to-dataurl';
+import { svgo } from "../SvgBackend/src/svgo.config";
+import SVGO from "svgo";
 
-const express = require("express");
 const app = express();
 const port = process.env.PORT || 3500;
+const svgoObject = new SVGO(svgo);
 
-// viewed at http://localhost:3500
-app.get("/", function(req, res) {
-    res.send("Again I Go Unnoticed");
+app.use(bodyParser.urlencoded({ extended: false }))
+app.use(bodyParser.json())
+
+app.get('/optimizeSvg',function(req,res){
```

Comment:
shouldnt this be a post endpoint, usually get requests dont have a body attached